File: Address-Objc/Address-Objc/ViewController.m
//
//  ViewController.m
//  Address-Objc
//
//  Created by Liu Chuan on 2018/10/4.
//  Copyright © 2018 LC. All rights reserved.
//

#import "ViewController.h"
#import "AddressViewController.h"

@interface ViewController ()

/**
 选择地址按钮
 */
@property (weak, nonatomic) IBOutlet UIButton *addressButton;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self.addressButton addTarget:self action:@selector(selectAddress) forControlEvents:UIControlEventTouchUpInside];
}

/**
 选择按钮点击事件
 */
-(void)selectAddress{
    AddressViewController *addressVC = [[AddressViewController alloc]init];
    UINavigationController *naVC = [[UINavigationController alloc]initWithRootViewController:addressVC];
    [self presentViewController:naVC animated:YES completion:nil];
}


@end

File: Address/Address/ViewController.swift
//
//  ViewController.swift
//  Address
//
//  Created by Liu Chuan on 16/9/11.
//  Copyright © 2016年 LC. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var addressButton: UIButton!  // 选择地址按钮

    override func viewDidLoad() {
        super.viewDidLoad()
        
        addressButton.addTarget(self, action: #selector(selectAddress), for: .touchUpInside)

        
    }
    
    // MARK: - 选择地址按钮点击事件
    @objc fileprivate func selectAddress() {
        
        let addressVC = AddressViewController()
        let naVC = UINavigationController(rootViewController: addressVC)
        self.present(naVC, animated: true, completion: nil)
    }
}

